core/commands/ls.go

```diff
@@ -48,6 +48,15 @@ the following format:
   <link base58 hash> <link size in bytes> <link name>
 
 The JSON output contains type information.
+The path can be a prefixless ref; in this case, we assume it to be an
+/ipfs ref and not /ipns.
+
+Example:
+
+    > ipfs file ls QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ
+    cat.jpg
+    > ipfs file ls /ipfs/QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ
+    cat.jpg
```

Comment:
I think this information is overkill.  Syntax for paths should be documented somewhere else.